packages/core/src/Stores/contract-store.js

```diff
@@ -259,7 +258,6 @@ export default class ContractStore extends BaseStore {
         if (contract_info) {
             if (isBarrierSupported(contract_type) && (barrier || high_barrier)) {
                 main_barrier?.updateBarriers(barrier || high_barrier, low_barrier);
-                main_barrier?.updateBarrierColor(is_dark_mode);
```

Comment:
@akmal-deriv same question: Currently, this function is executed when there is an ongoing contract with a barrier on the main Trading page or in Contract details.
What's the reason for removing this logic? What are we replacing it with? :)
Is it correct that we won't be updating the barrier color anymore via deriv-app, and it's gonna be done internally inside SmartCharts? I suppose we might wanna keep the ability to set the color from deriv-app, at least some part of it, like the main barrier color.

Reply:
Same answer, this is just a function to change barrier color for dark theme which is not used anymore, since the color is blue. It is been passed from deriv-app just like before.